package.json

```diff
@@ -60,22 +58,29 @@
     "jest-environment-jsdom": "^29.7.0",
     "jest-environment-node": "^29.7.0",
     "jest-junit": "^16.0.0",
+    "ng-packagr": "^17.3.0",
     "prettier": "^3.2.5",
     "react": "^18.2.0",
     "rollup": "^3.29.4",
     "rollup-plugin-dts": "^5.3.1",
+    "rxjs": "~7.8.0",
     "shx": "^0.3.4",
     "ts-jest": "^29.1.2",
     "ts-node": "^10.9.2",
+    "tslib": "^2.3.0",
     "typedoc": "^0.25.7",
     "typescript": "^4.7.4",
     "uuid": "^9.0.1"
   },
+  "overrides": {
+    "typescript": "^4.7.4"
```

Comment:
What is this for?

Reply:
`"jest-preset-angular": "^14.2.2"` requires typescript <= 5.2 but it does not really need it.
To have tests working in the whole npm workspace, this is needed to be able to use the "workspace ts" version instead of the one required. But it is not actually a problem for `jest-preset-angular`.
I also considered updating TS to 5.x but there were other libs involved then.